package Java_Exception_Handling.Java_Throw;

import java.io.IOException;

public class TestThrowsExample4 {

    public static void main(String[] args) throws IOException {
        TestDemo test = new TestDemo();
        test.method();
        System.out.println("Continue the program...");

    }
}
